packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -218,11 +227,13 @@ abstract class AnimatedWidgetBaseState<T extends ImplicitlyAnimatedWidget> exten
 /// A container that gradually changes its values over a period of time.
 ///
 /// This class is useful for generating simple implicit transitions between
-/// different parameters to [Container]. For more complex animations, you'll
-/// likely want to use a subclass of [Transition] or use an
-/// [AnimationController] yourself.
+/// different parameters to [Container] with its internal
+/// [AnimationController]. For more complex animations, you'll likely want to
+/// use a subclass of [Transition] or use your own [AnimationController].
 ///
-/// Properties that are null are not animated.
+/// The [AnimatedContainer] will automatically animate between the old and
+/// new values of properties when they change using the provided curve and
+/// duration. Properties that are null are not animated.
```

Comment:
maybe switch the second and third paragraphs? That way it says what it does before saying it's useful...